orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/render/JinjaRenderer.java

```diff
@@ -67,6 +69,10 @@ public JinjaRenderer(RenderedValueConverter renderedValueConverter, ObjectMapper
     jinja.setResourceLocator(new NoopResourceLocator());
     jinja.getGlobalContext().registerTag(new ModuleTag(this, pipelineTemplateObjectMapper));
```

Comment:
Module having a circular dependency on Renderer is sort of an inevitability for that individual tag, but I agree that these tags should always be there, since they are key to implementing the spec.